index.html

```diff
@@ -14,16 +14,53 @@
     <title>Awesome Book</title>
   </head>
   <body>
-    <h1>Awesome <span>Book</span>Store</h1>
-    <div class="displayBook"></div>
+    <header>
+      <h1>Awesome <span>Book</span>Store</h1>
+      <nav>
+        <ul>
+          <li class="list">List</li>
+          <li class="add-new-book">Add Contact</li>
+          <li class="contact">Contact</li>
+        </ul>
```

Comment:
- [ ] Like the previous reviewer suggested, please ensure that the links do exactly what they say. In this case, please change **Add Contact** to **Add Book**